docs/design/2021-04-26-lock-view.md

```diff
@@ -151,9 +155,25 @@ The locking key and `resource_group_tag` that comes from the `Context` of the pe
 
 The wait chain will be added to the `Deadlock` error which is returned by the `PessimisticLock` request, so that when deadlock happens, the full wait chain information can be passed to TiDB.
 
+### Configurations
+
+#### TiDB Config File `pessimistic-txn.tidb_deadlock_history_capacity`
+
+Specifies how many recent deadlock events each TiDB node should keep.
+Dynamically changeable via HTTP API.
+Value: 0 to 10000
+Default: 10
+
+#### TiDB Config File `pessimistic-txn.tidb_deadlock_history_collect_retryable`
```

Comment:
This will be a low priority development task and perhaps out of this sprint, depending on the time.